python_modules/libraries/dagster-docker/dagster_docker/container_context.py

```diff
@@ -88,7 +88,7 @@ def merge(self, other: "DockerContainerContext"):
             registry=other.registry if other.registry != None else self.registry,
             env_vars=[*self.env_vars, *other.env_vars],
             networks=[*self.networks, *other.networks],
-            container_kwargs=merge_dicts(other.container_kwargs, self.container_kwargs),
+            container_kwargs=merge_dicts(self.container_kwargs, other.container_kwargs),
```

Comment:
nit: i've been switching away from our random `merge_dicts` util to language native `{**x, **y}`

Reply:
Yes please ❤️

Reply:
done